### ISSUE TITLE
ArgumentNullException on placing order

### ISSUE THREAD

**Issue by rbasniak** [JonPSmith/EfCore.GenericBizRunner#5]
Hi,

I just made a new clone, and run the ExampleWebApp, but when I try to place an order, I get a `NullReferenceException `on the `PlaceOrderActions.BizAction` method, because the `CheckoutLineItems `property on the DTO is null.

By the way, your book is great! I think it's the best practical programming related book I ever read.

**Comment by JonPSmith** [comment]
Hi @rbasniak,

I'm sorry about that! I made a change just before release to make the examples more 'normal'. I used to get the books from the cookies, but I wanted it to get the lineitems from the form as that is the normal way to get data. I got the name of the LineItem part wrong  - it needed to be `CheckoutLineItems`. All fixed now and pushed. Please let me know if that fixed it and I will close this issue.

Thanks for the very kind comment about my book. I tried to write a book that I would want to read, with all the tricks and tips I could think of. 

PS. Have you seen my [EfCore.GenericServices](https://github.com/JonPSmith/EfCore.GenericServices) project? There is an [introduction article](https://www.thereformedprogrammer.net/genericservices-a-library-to-provide-crud-front-end-services-from-a-ef-core-database/) on my tech blog.

**Comment by rbasniak** [comment]
@JonPSmith it worked. Thanks a lot for the fast solution.

I didn't see that project yet, I found your blog last week, and after reading a couple of articles I didn't put a second thought on buying your book. I worked on a big project with EF Core that is finishing now, and you tries to solve the major problems I faced. I'm almost finishing, but there's too much to absorb :) But for sure I'll take a look at GenericServices and the testing library for EFCore.
